examples/raspberry_pi_inky_phat.rs

```diff
@@ -30,12 +30,13 @@ use std::{fs, io};
 const ROWS: u16 = 212;
 const COLS: u8 = 104;
 
+// More documentation for LUTs can be found at https://github.com/pimoroni/inky/blob/master/library/inky/inky.py#L95
```

Comment:
Good idea but can you make this a permalink (press `y` on the GitHub page) so that if the code changes the link will still point at the right thing.